interfaces/builtin/test_snapd.go

```diff
@@ -0,0 +1,99 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
```

Comment:
What's the point of this interface? I see it referenced from one of the test snaps below. How is that expected to work given that it won't be a part of non-test builds (due to file name).